lib/everypolitician/popolo.rb

```diff
@@ -52,6 +52,16 @@ def posts
       def memberships
         Memberships.new(popolo[:memberships])
       end
+
+      def legislative_periods
+        events.where(classification: 'legislative period').sort_by(&:start_date)
+      end
+      alias_method :terms, :legislative_periods
+
+      def current_legislative_period
+        legislative_periods[-1]
```

Comment:
Fixed in https://github.com/everypolitician/everypolitician-popolo/pull/38/commits/a579c2a41726a1622964148e85229c29706adc78